server/services/alexa/lib/alexa.onExecute.js

```diff
@@ -36,6 +50,35 @@ function onExecute(body) {
           f.type === DEVICE_FEATURE_TYPES.LIGHT.BINARY,
       );
       value = writeValues['Alexa.PowerController'](directiveName);
+      nameOfAlexaFeature = 'powerState';
+      break;
+    case 'Alexa.BrightnessController':
+      deviceFeature = deviceInMemory.features.find(
+        (f) => f.category === DEVICE_FEATURE_CATEGORIES.LIGHT && f.type === DEVICE_FEATURE_TYPES.LIGHT.BRIGHTNESS,
+      );
+      value = writeValues['Alexa.BrightnessController'](
+        directiveName,
+        get(body, 'directive.payload'),
+        deviceFeature.last_value,
+      );
+      nameOfAlexaFeature = 'brightness';
+      // if the brightness is set to 0, turn off the light
+      if (value === 0) {
+        controlPowerIfNeeded(deviceFeature.category, 0);
+      } else {
+        // if the brightness is set to something positive, make
+        // sure the light is on
+        controlPowerIfNeeded(deviceFeature.category, 1);
+      }
+      break;
+    case 'Alexa.ColorController':
```

Comment:
And maybe split each case in a dedicated file, it can grow quickly.

Reply:
We'll see in the future to split it if the file becomes too big, but for now let's keep it like that, it's more readable while it's small